api/search/views.py

```diff
@@ -668,6 +668,8 @@ def search_fields(self):
             'provider',
             'title',
             'tags',
+            'schoolType',
+            'studyDesign',
```

Comment:
Looks like these aren't needed here (in the Ticket, Brian says he can search by other metadata fields that aren't in the list). Not sure if their optional-ness makes them problematic to have